config/mysql.yaml.orig

```diff
@@ -0,0 +1,9 @@
+host: "127.0.0.1"
+port: 3307
+<<<<<<< HEAD
```

Comment:
need to remove residual traces of merge, such as "<<<<<<<< HEAD"

unmodified documents do not need to be submitted